build.sbt

```diff
@@ -6,7 +6,7 @@ lazy val microservice = Project("gvms-haulier-status", file("."))
   .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
   .settings(
     majorVersion := 0,
-    scalaVersion := "2.13.12",
+    scalaVersion := "3.4.2",
```

Comment:
can we all stick to one version going forward 3.5.0 unless i missed something ?

Reply:
As Ben suggested, we can leave it for now until the next dev ticket comes up.

Reply:
upgrade to 3.5.0 now.